Kirill_Kazimirov/02.Git.Local/README.md

```diff
@@ -0,0 +1,135 @@
+#Homework Assignment 1: Initializing a Local Repository
+    7  cd /home/kirill/courses/devops/sa.it-academy.by/Kirill_Kazimirov/02.Git.Local
```

Comment:
Please use code block for history sections 
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

Reply:
Fix it